.eslintignore

```diff
@@ -1,9 +1,7 @@
-node_modules
 .loom
-.vscode-test
+.next
+build
+build-internal
```

Comment:
I would prefer to keep these opinionated for now
```suggestion
/polaris-react/build
/polaris-react/build-internal
```

Reply:
Agreed, updated in other `.*ignore` files as well